libs/client-sdk/src/data_sources/polling_data_source.cpp

```diff
@@ -88,6 +88,10 @@ PollingDataSource::PollingDataSource(
     auto const& polling_config = std::get<
         config::shared::built::PollingConfig<config::shared::ClientSDK>>(
         data_source_config.method);
+    if (http_properties.Tls().VerifyMode() ==
```

Comment:
Could pass a logger dep to the `requester_` and have it do the logging instead. In any case, I just wanted to be sure people see a log message since this is pretty critical. Perhaps it should be at info or warn level.